### ISSUE TITLE
Question - how to backup a private repository from an organisation.

### ISSUE THREAD

**Issue by francoislauger** [josegonzalez/python-github-backup#104]
I, i'm a owner of an organization : [(https://github.com/CEDIA-models)]  and I want to backup issues from a single private repository. 

My idea was to log with my account(francoislauger)  and acces the organization with directory : 
`github-backup -u USER francoislauger  -O -P -R CEDIA-models/REPO  --issues --issue-comments
`
I get a prompt for a password but after that I get that 
```
Requesting https://api.github.com/user?per_page=100&page=1
API request returned HTTP 403: Forbidden
```
I'm sure i'm doing something wrong but not sure what.

**Comment by whwright** [comment]
I believe the last argument should only be the organization name. If you want to back up a certain repo, you need to use

```
  -R REPOSITORY, --repository REPOSITORY
```

So maybe something like `github-backup -u ${USER} -p ${PASSWORD} -O -P -R ${REPO} ${ORG}

**Comment by francoislauger** [comment]
@whwright If I do that with CEDIA-models for ${ORG} I get `github-backup: error: unrecognized arguments: CEDIA-models`

This is the exact command : `github-backup -u USER francoislauger  -P   --issues --issue-comments -R repo1 -O CEDIA-models`

**Comment by whwright** [comment]
I think this might be an order or arguments thing? I did this with success on an org/repo I have access to

```
↪ github-backup -t ${GITHUB_TOKEN} --issues --issue-comments -R ${REPO} -P ${ORGANIZATION}
```

**Comment by francoislauger** [comment]
Now with : 
`github-backup -t ${GITHUB_TOKEN}  -O  --issues --issue-comments CEDIA-models`

I can backup public folder of CEDIA-models. However when I try to backup private repository with: 

`github-backup -t  ${GITHUB_TOKEN}   -O  --issues --issue-comments -P -R simul  CEDIA-models`

I got this warning:
`Authenticated user is different from user being backed up, thus private repositories cannot be accessed`

The token loggeg to  `francoislauger` and I have full acces to CEDIA-models. I tried with and without the -O. Any idea?

**Comment by whwright** [comment]
This is a bug that was fixed in #103 and is in master branch, but has not been released to pypi.

**Comment by francoislauger** [comment]
@whwright Thank, I reinstall from github and it's working. 

Note, the warning (`Authenticated user is different from user being backed up, thus private repositories cannot be accessed`) still there but seem to be ignore.

**Comment by josegonzalez** [comment]
Released 0.22.0.

**Comment by whwright** [comment]
I think this can close now
